.github/workflows/linux.yml

```diff
@@ -37,12 +43,12 @@ jobs:
           # FIXME: Edit when KLEE is rebased to latest upstream.
           - {os: ubuntu-20.04, llvm: 11, compiler: clang, klee: 'no-klee'}
           - {os: ubuntu-20.04, llvm: 12, compiler: clang, klee: 'no-klee'}
-          - {os: ubuntu-20.04, llvm: 13, compiler: clang, klee: 'no-klee'}
+          - {os: ubuntu-22.04, llvm: 13, compiler: clang, klee: 'no-klee'}
 
     runs-on: ${{matrix.os}}
     steps:
       - name: Checkout Symbiotic and submodules
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
```

Comment:
Why do we need this change?

Reply:
Well, there's no reason to not to use the latest version of this action.